pom.xml

```diff
@@ -50,6 +50,21 @@
     </plugins>
   </build>
   <dependencies>
+    <dependency>
+      <groupId>com.azure.resourcemanager</groupId>
+      <artifactId>azure-resourcemanager</artifactId>
+      <version>2.9.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.4.0</version>
```

Comment:
given how old the PR is now, it would be good to refresh this to much newer versions of libraries.

Reply:
According to your suggestion, fixed the newer version of libraries in the new version.